packages/api/src/controllers/orchestrator.ts

```diff
@@ -1,11 +1,18 @@
+import { Request } from "express";
 import Router from "express/lib/router";
 
 const app = Router();
 
-const getOrchestrators = async (req, res, next) => {
-  const orchestrators = await req.getOrchestrators(req);
+const getOrchestrators = async (req: Request, res, next) => {
+  const orchestrators = [];
+  for (const getter of req.orchestratorsGetters) {
+    const orchs = await getter();
+    orchestrators.push(...orchs);
+  }
 
-  return res.json(orchestrators.map(({ address }) => ({ address })));
+  return res.json(
+    orchestrators.map(({ address, score }) => ({ address, score }))
```

Comment:
Should the `defaultScore` be here instead of in the `regionOrchestratorsGetter`? So that any getter might return a non-scored orch and we always give it a default here.
```suggestion
    orchestrators.map(({ address, score = defaultScore }) => ({ address, score }))
```

Reply:
fixed